test/sanity/validate-modules/validate-modules

```diff
@@ -851,6 +853,30 @@ class ModuleValidator(Validator):
 
         return False
 
+    def _check_for_subprocess(self):
+        for child in self.ast.body:
+            if isinstance(child, ast.Import):
```

Comment:
I'd rather remove this.  We also wouldn't catch the problem where the import does not happen at the root level, such as if placed in a `try/except`.

We are already going parse through hundreds of lines, looping the ast, it's likely faster to just search in the text.